includes/theme_form_element_label.inc

```diff
@@ -6,53 +6,14 @@
  */
 
 /**
- * Returns HTML for a form element label.
- *
- * - Wrap radio/checkbox labels in a `<span>`.
- *
- * See drupal/includes/form.inc and theming/theming.theme.inc.
+ * Preprocess variables for the form_element_label theme function.
  */
-function campaignion_foundation_form_element_label($variables) {
-  $element = $variables['element'];
-
-  // This is also used in the installer, pre-database setup.
-  $t = get_t();
-
-  // If title and required marker are both empty, output no label.
-  if ((!isset($element['#title']) || $element['#title'] === '') && empty($element['#required'])) {
-    return '';
-  }
-
-  // If the element is required, a required marker is appended to the label.
-  $required = !empty($element['#required']) ? theme('form_required_marker', array(
-    'element' => $element,
-  )) : '';
-  $title = filter_xss_admin($element['#title']);
-  $attributes = $element['#label_attributes'] ?? array();
-
-  // Style the label as class option to display inline with the element.
+function campaignion_foundation_preprocess_form_element_label(&$variables) {
```

Comment:
I'd not expect a preprocess hook in `themes_xxx.inc` – `template.php` probably fits better now.

Reply:
Great, thank you!

I think it would be helpful to keep the original comments to remember why we did this. 

> Wrap radio/checkbox labels in a `<span>`.

Just realizing the label_for thing is actually part of the original Drupal `theme_form_element_label`, was it explicitly excluded from `theming_preprocess_form_element_label`? I don’t think we’re actively using it anywhere (but might be relevant for other Drupal code)?

Reply:
> Wrap radio/checkbox labels …

Ok, I’ve added this to the comment above the preprocess function.

> #label_for

This seemed too useful to be part of the default implementation. I’ve found it there and just copied to the theming module instead (and removed it here).

Reply:
Thank you!